packages/amplify_auth_plugin_interface/lib/amplify_auth_plugin_interface.dart

```diff
@@ -94,19 +94,19 @@ abstract class AuthPluginInterface extends AmplifyPluginInterface {
   }
 
   Future<UpdateUserAttributeResult> updateUserAttribute(
-      {@required UpdateUserAttributeRequest request}) {
+      {UpdateUserAttributeRequest? request}) {
```

Comment:
Same as above (I would expect this to be required)

Reply:
Same as above.